### DISCUSSION TITLE
microwin + ventoy

### DISCUSSION THREAD

**Discussion by foolofdestiny** [ChrisTitusTech/winutil#1707] [Q&A]
ive heard of ventoy because of linux use
for microwin. do i need to install ventoy and create a bootable usb with it?
"Copy  this image to your Ventoy USB Stick"

there is the option [ ]"copy to ventoy"  -  the pop up tooltip say copy to usb with **_labeled_** ventoy

so curious what role ventoy plays in this proccess. ?

PS..'feature request' clarfiy (if any) this in the winutil>microwin
and link (if theres) docs or references for microwin

Ty.

**Comment by ModernTTY** [comment]
You need to make a Ventoy bootable usb drive, then copy the Microwin ISO to the Ventoy drive. Instructions [here](https://www.ventoy.net/en/doc_start.html)
